tests/coalaTest.py

```diff
@@ -179,3 +179,19 @@ def test_logged_error_causes_non_zero_exitcode(self):
             )
 
             assert exitcode == 1
+
+    def test_coala_no_color(self):
+        configure_logging(color=False)
+        with bear_test_module(), \
+                prepare_file(['#fixme'], None) as (lines, filename):
+            retval, stdout, stderr = execute_coala(
+                             coala.main,
+                             'coala', '-c', os.devnull,
+                             '-f', re.escape(filename),
+                             '-b', 'LineCountTestBear')
+            self.assertIn('This file has 1 lines.',
+                          stdout,
+                          'The output should report count as 1 lines')
```

Comment:
I'm making stderr check more restrictive. I think timestamps are also there which can't be checked easily. IMO stdout check is good enough.